va/va.h

```diff
@@ -4730,6 +4730,56 @@ typedef struct _VAPictureHEVC
  */
 #define VA_PICTURE_HEVC_RPS_LT_CURR             0x00000040
 
+typedef enum{
+    VACopyObjectSurface = 0,
+    VACopyObjectBuffer  = 1,
+} VACopyObjectType;
+
+typedef struct _VACopyObject {
+    VACopyObjectType  obj_type;    // type of object.
+    union
+    {
+        VASurfaceID surface_id;
+        VABufferID  buffer_id;
+    } object;
+
+    uint32_t    va_reserved[VA_PADDING_MEDIUM];
+} VACopyObject;
+
+#define VA_COPY_SYNC_BLOCK    0x0
+#define VA_COPY_SYNC_NONBLOCK 0x1
+
+#define VA_COPY_MODE_BALANCE      0x0
+#define VA_COPY_MODE_PERF         0x1
+#define VA_COPY_MODE_POWER_SAVING 0x2
+
+typedef union _VACopyOption{
+    struct {
+        /** \brief va copy synchronization, the value should be VA_COPY_SYNC_XXX
+         * if it equal VA_COPY_SYNC_NONBLOCK , sync handle should be retrieved.
```

Comment:
@XinfengZhang , could you remind me (and better to describe in va.h) what sync handle is and how to retrieved

Reply:
thanks , I removed it, we should use vaSyncSurface or vaSyncBuffer to handle it